cpp/src/Node.cpp

```diff
@@ -498,6 +501,42 @@ void Node::AdvanceQueries
 				}
 				break;
 			}
+			case QueryStage_Groups:
```

Comment:
Wouldn't the QueryStage_Association get us this info when it completes?
(Groups == Associations right?)